docs/running-node-manual/on.md

```diff
@@ -0,0 +1,79 @@
+# Running an Observer Node manually
+
```

Comment:
It looks like most of the node types (ON, VN, Sentry, Seed) share the same steps. Maybe we can create a generic instruction full-node.md, and concrete instructions for every nodes type can just referencing it, specifying the difference (usually config params).